test/core/ExamplesUnitTests/TribitsExampleProject_Tests.cmake

```diff
@@ -2801,12 +2801,13 @@ tribits_add_advanced_test( TribitsExampleProject_compiler_flags
   OVERALL_WORKING_DIRECTORY  TEST_NAME
   OVERALL_NUM_MPI_PROCS  1
   EXCLUDE_IF_NOT_TRUE  IS_REAL_LINUX_SYSTEM  COMPILER_IS_GNU
-    ${PROJECT_NAME}_ENABLE_Fortran
+    ${PROJECT_NAME}_ENABLE_Fortran  NINJA_EXE
```

Comment:
You choose to use Ninja to get verbose output with cmake 3.23. Do we have the also the verbose output with Ninja and cmake < 3.23 ?

Reply:
I have no idea but it does not really matter in this context.